.github/workflows.src/tests.inc.yml

```diff
@@ -432,12 +432,7 @@
 
         # Validate Rust extensions
         for ext in ./build/rust_extensions/edb/*.so; do
-          ldd "$ext" || exit 1
-        done
-
```

Comment:
Why?

Reply:
I thought I'd add a test for all the stuff we're restoring from cache -- it seems unlikely that the extensions will break but I suppose it's possible if there's some weirdness going on with openssl libraries moving around, and it would be pretty annoying to track down if it did.

I can take that out.

Reply:
Oh I had just noticed the removal, not the addition, in the "[Skip validation of cython extensions](https://github.com/edgedb/edgedb/pull/7875/commits/505779b1674b86fe96e809a083cc003c21be43a9)" commit